### ISSUE TITLE
SQITCH_ORIG_FULLNAME on windows

### ISSUE THREAD

**Issue by lastlink** [sqitchers/docker-sqitch#33]
continue of conversation from #32 

Running git bash on windows is like a 50 line username info that needs to be filtered down on windows.

**Comment by lastlink** [comment]
@theory are you trying to correct the fullname on windows cmd and git bash?


I'm not fully understanding what a fullname is supposed to be if it's not the username? is it like the userdirectory/username?

that command from https://stackoverflow.com/questions/7809648/get-display-name-of-current-windows-domain-user-from-a-command-prompt/34647284#34647284 that you shared would only tweak the cmd .bat file and not the git bash which would need to be tweaked.

**Comment by theory** [comment]
Yeah, the idea is to have a reasonable default value for the full name of the user who hasn't set `sqitch config --user user.name`.
